src/vendor/wordpress/styles/_core-blocks.scss

```diff
@@ -157,6 +157,17 @@ $wp-button-gap: size.$spacing-gap-button-group-default;
     margin-block-start: 0; // 1
     max-inline-size: calc(50% - 0.5em); // 2
   }
+
+  /// Apply border only to image
+  ///
+  /// 1. We're using 1px borders here rather than size.$edge-small because it
+  ///    was causing subpixel rendering errors leaving a slight gap between the
+  ///    image and the border.
+  &.is-style-outlined img {
```

Comment:
@spaceninja Can you help me understand why the code uses a "state" `is-` prefix here? I was expecting a modifier class (e.g., `.wp-block-image--outlined`). 

I might be missing something obvious. I haven't had a chance to create new blocks or CSS for blocks yet. 🙂

Reply:
Yep — we don't have control over it. We're using WordPress's native "block styles" which let you name your style (in this case "outlined" and it automatically adds a class, in this case `is-style-outlined`.

Since we're dealing with extending a built-in block, our options are somewhat limited.